src/Common/src/Common.Hosting/HostBuilderExtensions.cs

```diff
@@ -13,7 +13,9 @@ public static class HostBuilderExtensions
     public const string DefaultUrl = "http://*:8080";
 
     /// <summary>
-    /// Configure the application to listen on port(s) provided by the environment at runtime. Defaults to port 8080.
+    /// Configure the application to listen on port(s) provided by the environment at runtime. Specifically it adds ports and/or urls from the following
+    /// environment variables: PORT, SERVERPORT, ASPNETCORE_URLS and in Kubernetes $"{appName}_SERVICE_PORT_HTTP") where appName is the prefix of HOSTNAME.
+    /// It also opens the application on the managementPort is expected to be used in conjuction with the ManagementPortMiddlware. Defaults to port 8080.
```

Comment:
```suggestion
    /// It also opens the application on the management port, which is expected to be used in conjunction with <see cref="ManagementPortMiddleware" />. Defaults to port 8080.
```

Reply:
The cref is to a type thats not available in this project. Other than that thanks - I will fix it as suggested.